doc/source/index.rst

```diff
@@ -26,7 +26,12 @@ Pyorbital has a module for parsing NORAD TLE-files
     >>> tle.inclination
     99.043499999999995
 
-If no path is given pyorbital tries to read the earth observation TLE-files from celestrak.com
+If no path is provided pyorbital first tries to read any local TLE files in the
+directory given by the environment variable :envvar:`TLES`. If this variable is not
+set Pyorbital will try get the earth observation TLE files over the internet
+from celestrak.com. Note this downloading only happens if no
+specific TLE file is provided or if the :envvar:`TLES` environment variable is not set.
```

Comment:
Forgive the ignorance, but how does it search for other TLE files in this directory? What is the expected naming scheme? Or are TLEs known to have a specific naming scheme?